### PR TITLE
fix: [OIP-620]: Add No Auth option to Authentication options in Splunk connector

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

Pasted Graphic 4

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by ishantmehtaharness — DISMISSED** [review]
Looks Good

File: src/modules/35-connectors/components/CreateConnector/SplunkConnector/CreateSplunkConnector.constants.ts [hunk 1]
```diff
@@ -10,5 +10,9 @@ export const getAuthOptions = (getString: UseStringsReturn['getString']): Select
   {
     label: getString('connectors.bearerToken'),
     value: AuthTypes.BEARER_TOKEN
+  },
```
Comment (sunnykesh-harness) [inline src/modules/35-connectors/components/CreateConnector/SplunkConnector/CreateSplunkConnector.constants.ts:13]: Please check once while editing and while moving across steps  does this auth option gets preselected if it is applicable.

**Review by sunnykesh-harness — COMMENTED** [review]
Also noticed a major UX miss, in left section connector name is missing below the icon. Please refer to other connectors.

File: src/modules/35-connectors/components/CreateConnector/SplunkConnector/CreateSplunkConnector.constants.ts (hunk 1 above)
Comment (praneshg239) [reply src/modules/35-connectors/components/CreateConnector/SplunkConnector/CreateSplunkConnector.constants.ts:13]: Yes, tested it.

**Comment by praneshg239** [comment]
> Also noticed a major UX miss, in left section connector name is missing below the icon. Please refer to other connectors.

image

Added the title below the connector icon

**Review by sunnykesh-harness — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**